modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -866,6 +866,10 @@ class Session implements ISession {
         config.navigate('nextflow.enable.moduleBinaries', false) as boolean
     }
 
+    boolean failOnIgnore() {
+        config.navigate('workflow.failOnIgnore', false) as boolean
```

Comment:
```suggestion
        config.navigate('nextflow.failOnIgnore', false) as boolean
```

Reply:
Check my comment https://github.com/nextflow-io/nextflow/pull/5071#issuecomment-2174019219

Reply:
It should go in this config scope: https://nextflow.io/docs/latest/config.html#scope-nextflow

Reply:
Disagree

Reply:
I'm not talking about feature flags, I'm talking about the `nextflow` config scope which contains the publish retry settings. It makes sense to group them together

Reply:
yeah, that was somehow temporary. We are going to deprecated `publishDir`, therefore those settings will likely go under `workflow.output.retryPolicy`.